Makefile

```diff
@@ -79,3 +79,8 @@ clean:
 	@find . -type d -name '*pytest_cache*' -exec rm -rf {} +
 	@find . -type f -name "*.py[co]" -exec rm -rf {} +
 	@find . -type d -name '*.ipynb_checkpoints' -exec rm -r {} +
+
+.PHONY: serve
+serve:
```

Comment:
How can someone run the server if they install Draco via pip? We should document that.